docs/en/index.md

```diff
@@ -71,29 +88,25 @@ class SendNewslettersCommand extends Command
 }
 ```
 
-The configure method is to name the command and specify arguments and options.
-They have a lot of options and you can read about them in Symfony Documentation.
 
-When the command is executed, the execute method is called with two arguments.
-First one is command input, which contains all the parsed arguments and options.
-The second one is command output which should be used to provide feedback to the developer which ran the command.
+In `__construct`, we setup the name of the command, then set some description and set all dependencies.
```

Comment:
So I do constructor dependencies as a matter of religion. :)

When you override the method, autogenerated stub [contains parameter $name](https://github.com/symfony/console/blob/master/Command/Command.php#L50).
So I tried setting the name directly there, and it worked, so I figured that's an approach, which is there for some reason. [for a very, very, very long time](https://github.com/symfony/console/blob/2.0/Command/Command.php#L46), and 3.0 did not remove it.

Also, I don't like using arbitraty `configure()` for no reason, as it is [called from constructor](https://github.com/symfony/console/blob/master/Command/Command.php#L62) and these days, it serves no purpose other than BC.
[Admittedly, it was useful in first versions of console](https://github.com/symfony/console/blob/2.0/Command/Command.php#L63) - since those really "prepped" the command for use.

**Neither of which is reflected in Symfony documentation.** I suppose I should create a PR for SF/console :)